From e27f153f14461d6efa741e302167122e29da7b58 Mon Sep 17 00:00:00 2001
From: John Regan <john@jrjrtech.com>
Date: Sun, 23 Apr 2023 17:05:41 -0400
Subject: [PATCH 06/12] UPSTREAM: Add missing libtsm_deps to pixman renderer

I had installed libtsm-4.0.2 to a non-standard directory and got an error while compiling - since libtsm_deps wasn't listed as a dependency, the required CFLAGS weren't being used while compiling.
---
 src/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/meson.build b/src/meson.build
index c18759f..4ac845b 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -219,7 +219,7 @@ if enable_renderer_pixman
       'kmscon_mod_pixman.c',
     ],
     name_prefix: '',
-    dependencies: [pixman_deps, shl_deps],
+    dependencies: [libtsm_deps, pixman_deps, shl_deps],
     install: true,
     install_dir: moduledir,
   )
-- 
2.47.0

